packages/ui/build/plugins/append-component-css.ts

```diff
@@ -27,15 +27,18 @@ export const appendComponentCss = createDistTransformPlugin({
 
   dir: (outDir) => `${outDir}/src/components`,
 
-  transform: (componentContent, path) => {
+  transform (componentContent, path) {
```

Comment:
Yes, because we provide an `outDir` value to the `transform` function.

Passing the `outDir` value to the function:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/29167241/229103282-41ae3e81-54fe-4d7d-ba8b-29af95914bae.png">

Getting value by `this.outDir`:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/29167241/229103946-4efd8320-6337-418e-8c96-deb07ed17d98.png">